pkg/assetvalidator/validateAssets.go

```diff
@@ -9,22 +9,42 @@ import (
 	"github.com/asecurityteam/nexpose-asset-producer/pkg/domain"
 )
 
+// NexposeScanType is a type alias to signify type of scan according to Nexpose
+type NexposeScanType string
+
+const (
+	automated NexposeScanType = "Automated"
+	manual    NexposeScanType = "Manual"
+	scheduled NexposeScanType = "Scheduled"
+	agent     NexposeScanType = "Agent"
+)
+
+// ScanType is a type alias that generifies a scan to either local or remote
+// The purpose is to not expose Nexpose specifics further down the pipeline
+type ScanType string
+
+const (
+	local   ScanType = "local"
+	remote  ScanType = "remote"
+	unknown ScanType = "unknown"
```

Comment:
task: given the logic flow below, the "unknown" value is not needed. It will never be actually used, since assets with "unknown" scan types get appended to the error list, and not the list of assets to be produced.